### PR TITLE
fix(nginx archive recipe): Fixes to various configuration files.

### LINKED ISSUE #3423: [Bug] building Nginx + Image Archive recipe fails
### description

I am following[ the guide ](https://v3-docs.ohif.org/deployment/nginx--image-archive/#cors-issues)to build ohif along with orthanc and nginx proxy. when I run `docker-compose up --build` from inside `Viewers/platform/viewer/.recipes/OpenResty-Orthanc` I get this error:
```
=> ERROR [builder 17/18] RUN yarn install                                                      0.5s
------                                                                                               
 > [builder 17/18] RUN yarn install:                                                                 
#0 0.358 yarn install v1.22.5                                                                        
#0 0.425 $ node preinstall.js                                                                        
#0 0.452 internal/modules/cjs/loader.js:818                                                          
#0 0.452   throw err;                                                                                
#0 0.452   ^                                                                                         
#0 0.452                                                                                             
#0 0.452 Error: Cannot find module '/usr/src/app/preinstall.js'
#0 0.452     at Function.Module._resolveFilename (internal/modules/cjs/loader.js:815:15)
#0 0.452     at Function.Module._load (internal/modules/cjs/loader.js:667:27)
#0 0.452     at Function.executeUserEntryPoint [as runMain] (internal/modules/run_main.js:60:12)
#0 0.452     at internal/main/run_main_module.js:17:47 {
#0 0.452   code: 'MODULE_NOT_FOUND',
#0 0.452   requireStack: []
#0 0.452 }
#0 0.461 error Command failed with exit code 1.
#0 0.462 info Visit https://yarnpkg.com/en/docs/cli/install for documentation about this command.
------
failed to solve: process "/bin/sh -c yarn install" did not complete successfully: exit code: 1
```
as far as I can tell, this error means that `preinstall.js` file is missing. this file exists inside `platform/` which docker does copies!
### Steps to Reproduce

1. switch to v3-stable branch
2. cd to `Viewers/platform/viewer/.recipes/OpenResty-Orthanc`
3. run `docker-compose up --build`

### The current behavior

docker fails with the above error.

### The expected behavior

I expect getting an image containing ohif that is linked to a local orthanc server.

### OS

tried this on macos ventura and ubuntu 22.10

### Node version

16.20.0

### Browser

-

### PR DESCRIPTION
**Pull request by jbocce**
### Context

Fixes #3423

Note that the OpenResty-Orthanc-Keycloak recipe is still broken, but some changes were made for it.

### Changes & Results

Various changes to docker, nginx and OHIF configuration files.

### Testing

Follow the [nginx-image-archive](https://deploy-preview-3624--ohif-platform-docs.netlify.app/deployment/nginx--image-archive) recipe in the OHIF docs.

#### PR

- [x] My Pull Request title is descriptive, accurate and follows the
  semantic-release format and guidelines.

#### Code

- [x] My code has been well-documented (function documentation, inline comments,
  etc.)

#### Public Documentation Updates

- [x] The documentation page has been updated as necessary for any public API
  additions or removals.

#### Tested Environment

- [x] OS: Windows 11 
- [x] Node version: 16.14.0 
- [x] Browser: Chrome 116.0.5845.111
  

[blog]: https://circleci.com/blog/triggering-trusted-ci-jobs-on-untrusted-forks/
[script]: https://github.com/jklukas/git-push-fork-to-upstream-branch

### CODE REVIEW

**Review by sedghi — APPROVED** [review]
Looks great, fixed it for ARM

File: platform/app/.recipes/OpenResty-Orthanc-Keycloak/dockerfile [hunk 1]
```diff
@@ -23,20 +23,35 @@
...
+FROM node:16.15.0-slim as builder
 
 RUN mkdir /usr/src/app
 WORKDIR /usr/src/app
 
 ENV APP_CONFIG=config/docker_openresty-orthanc-keycloak.js
 ENV PATH /usr/src/app/node_modules/.bin:$PATH
 
+# Copy Files
+COPY .docker /usr/src/app/.docker
```
Comment (sedghi) [inline platform/app/.recipes/OpenResty-Orthanc-Keycloak/dockerfile:35]: is there any folder that we don't need to copy? maybe we say copy all excluding this and that?

Comment (jbocce) [reply platform/app/.recipes/OpenResty-Orthanc-Keycloak/dockerfile:35]: I think the list to exclude is bigger than the list to include.

Comment (jbocce) [reply platform/app/.recipes/OpenResty-Orthanc-Keycloak/dockerfile:35]: Also, I couldn't find a way to exclude files using the `COPY` command. But... I think I may have found a way using the `.dockerignore` file. Stay tuned.

**Review by sedghi — APPROVED** [review]

**Pull request merged**